src/main/java/frc/robot/sensors/ArmSensor.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.sensors;
+
+import static edu.wpi.first.units.Units.Millimeter;
+
+import au.grapplerobotics.ConfigurationFailedException;
+import au.grapplerobotics.LaserCan;
+import edu.wpi.first.units.measure.Distance;
+import frc.robot.Hardware;
+
+public class ArmSensor {
+
+  private final LaserCan mainSensor;
+
+  public ArmSensor() {
+    mainSensor = new LaserCan(Hardware.MAIN_ARM_SENSOR);
+    ConfigureSensor(mainSensor);
+  }
+
+  private void ConfigureSensor(LaserCan Sensor) {
+    try {
+      Sensor.setRangingMode(LaserCan.RangingMode.SHORT);
+      Sensor.setRegionOfInterest(new LaserCan.RegionOfInterest(8, 8, 16, 16));
+      Sensor.setTimingBudget(LaserCan.TimingBudget.TIMING_BUDGET_33MS);
+    } catch (ConfigurationFailedException e) {
+      System.out.println("Configuration Failed! " + e);
+    }
+  }
+
+  public Distance getSensorDistance() {
+    LaserCan.Measurement measurement = mainSensor.getMeasurement();
+    if (measurement != null && measurement.status == LaserCan.LASERCAN_STATUS_VALID_MEASUREMENT) {
+      return Millimeter.of(measurement.distance_mm);
+    } else {
+      return Millimeter.of(-1);
```

Comment:
Is it useful to return a different result for measurement == null (-2) vs. LaserCan.LASERCAN_STATUS_VALID*  (-1)?

E.g. if the configuration failed vs. an invalid measurement?